src/anyio/_backends/_asyncio.py

```diff
@@ -2082,13 +2082,23 @@ async def _run_tests_and_fixtures(
             tuple[Awaitable[T_Retval], asyncio.Future[T_Retval]]
         ],
     ) -> None:
+        from _pytest.outcomes import OutcomeException
+
         with receive_stream, self._send_stream:
             async for coro, future in receive_stream:
                 try:
                     retval = await coro
+                except CancelledError:
```

Comment:
Do you want to extract and pass on the cancel message?

Reply:
Hm, might be a good idea.